src/test/functional/test.js

```diff
@@ -0,0 +1,37 @@
+import { ReactSelector, waitForReact } from 'testcafe-react-selectors';
+import UserInfo from '../../stores/user-info';
+
+fixture`App tests`
+    .page(`http://localhost:3000`)
+    .beforeEach(async () => {
+        await waitForReact();
+    });
+
+test('Update user', async t => {
+    const userInfo       = new UserInfo(
+        {
+            name:  'Viktoria',
+            phone: '+79500418181'
+        }
+    );
+    const incorrectPhone = 'incorrect';
```

Comment:
This var we use in only one place, so we can use it inline in `typeText` action.
It's ok.